docs/authentication/embedded-sdk-add-authentication.mdx

```diff
@@ -179,7 +179,7 @@ codeChallenge = crypto
 The [authenticate call returns](/docs/embedded-reference-authenticate#returns) an object that contains a **redirectURL** to which your app should redirect the user to complete the OIDC flow.  
 Keeping with the OIDC specifications, this URL includes the **code** and **state** parameters as query parameters.
 
-Calling the `/token` endpoint is the second step in the authorization flow. In this call, your app sends the **code** as part of an OAuth call to exchange it for an access token.  
+Calling the `/token` endpoint is the second step in the authorization flow. In this call, your app sends the **code** as part of an OAuth call to exchange it for an access token and id token.  
```

Comment:
This is where I actually meant to ask about the `openid` scope value. 😂 

the `/token` endpoint is where I get the id_token by passing scope=openid in the body of my request.

Here's where I make the request (lines 52-54) in WHNVR as an example:
https://github.com/sam-huckaby/whnvr/blob/main/lib/auth/auth.ml

Reply:
I added more details for the openid scope (on the /authorize call). Let me know if that helps.